ntbs-service/Services/NotificationChangesService.cs

```diff
@@ -44,8 +44,8 @@ public async Task<IEnumerable<NotificationHistoryListItemModel>> GetChangesList(
                 .SelectMany(GetCanonicalLogs)
                 .ToList();
 
-            // We do not detail out the changes that happened before submission  to avoid unnecessary page clutter
-            var auditLogsToShow = SkipDraftEdits(auditLogs);
+            // We do not detail out the changes that happened before submission or print events to avoid unnecessary page clutter
+            var auditLogsToShow = SkipDraftEdits(auditLogs).Where(a => a.EventType != AuditService.PRINT_EVENT);
```

Comment:
Conceptually, I think it would be better to filter out print events in the same place that we filter out read events - which is in the `AuditService` in `GetWriteAuditsForNotification`.

Reply:
I have moved the filtering to `GetWriteAuditsForNotification`. This meant that the way I was testing the filtering wasn't going to work so I have created some tests for the audit service now too.